pyproject.toml

```diff
@@ -105,9 +105,10 @@ cmd = "pip uninstall temporalio -y"
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
-log_cli = true
-log_cli_level = "INFO"
-log_cli_format = "%(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)"
+# Do not use log_cli since this shows logging for all tests, not just the ones
+# that failed. Instead, show all logs for failed tests at the end.
+log_level = "DEBUG"
+log_format = "%(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)"
```

Comment:
Near the bottom of the README there is:

```markdown
For example, here's how to run a single test with debug logs on the console:

    poe test -s --log-cli-level=DEBUG -k test_sync_activity_thread_cancel_caught
```

I admit I use this a _lot_ during development on passing tests (I put this at the bottom of the README on .NET and Ruby too because I forget the args going from one to the other). Will this still work or does this need to be updated now to enable CLI logging?

Reply:
That still works: 
<img width="1725" alt="image" src="https://github.com/user-attachments/assets/d8004b2b-66e8-4ab6-881f-fb75446fef62" />